patches/expo-image+2.3.2.patch

```diff
@@ -1,12 +1,11 @@
 diff --git a/node_modules/expo-image/android/src/main/java/expo/modules/image/okhttp/ExpoImageOkHttpClientGlideModule.kt b/node_modules/expo-image/android/src/main/java/expo/modules/image/okhttp/ExpoImageOkHttpClientGlideModule.kt
-index 071907c..1995904 100644
+index 071907c..c4bdbc8 100644
 --- a/node_modules/expo-image/android/src/main/java/expo/modules/image/okhttp/ExpoImageOkHttpClientGlideModule.kt
 +++ b/node_modules/expo-image/android/src/main/java/expo/modules/image/okhttp/ExpoImageOkHttpClientGlideModule.kt
-@@ -24,6 +24,17 @@ class GlideUrlWithCustomCacheKey(
+@@ -24,6 +24,15 @@ class GlideUrlWithCustomCacheKey(
    headers: Headers?,
    private val cacheKey: String
  ) : GlideUrl(uri, headers) {
-+
 +  companion object {
```

Comment:
This old file is not needed anymore.
We removed it on #6347 here https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/f741be331ea18f2f07912597e29bfe5d86a143ff/android/app/src/main/java/chat/rocket/reactnative/networking/ExpoImageClient.java

We kept it there with an `_` suffix as a mistake.
It was not doing anything, but if we add it again, it's likely to create a new bug.